relayer/pkg/chainlink/txm/txm.go

```diff
@@ -35,19 +42,19 @@ type starktxm struct {
 	lggr    logger.Logger
 	done    sync.WaitGroup
 	stop    chan struct{}
-	queue   chan types.Transaction
+	queue   chan Tx
 	ks      keys.Keystore
 	cfg     Config
 
 	// TODO: use lazy loaded client
-	client    types.Provider
-	getClient func() (types.Provider, error)
+	client    *starknet.Client
```

Comment:
can this be an interface instead?

that way we can plug in mocked clients, etc

Reply:
ohhhh I see...the `Account` interface now requires the gateway client not an interface anymore